package.json

```diff
@@ -27,16 +27,19 @@
     "*.d.ts"
   ],
   "scripts": {
-    "archive-storybook": "archive-storybook",
-    "build-archive-storybook": "build-archive-storybook",
+    "archive-storybook:playwright": "archive-storybook",
+    "build-archive-storybook:playwright": "build-archive-storybook",
+    "archive-storybook:cypress": "CHROMATIC_ARCHIVE_LOCATION=./tests/cypress/downloads archive-storybook",
+    "build-archive-storybook:cypress": "CHROMATIC_ARCHIVE_LOCATION=./tests/cypress/downloads build-archive-storybook",
     "clean": "rimraf ./dist",
     "prebuild": "yarn clean",
     "build": "tsup",
     "build:watch": "yarn build --watch",
     "test:unit": "jest",
     "test:playwright": "playwright test",
-    "test:cypress": "cypress run --project tests",
-    "test:server": "node tests/playwright/server",
+    "test:cypress": "start-server-and-test test:server 3000 test:do-cypress",
```

Comment:
I'm using this `start-server-and-test` package because Cypress [says](https://docs.cypress.io/guides/continuous-integration/introduction#Boot-your-server) to not just run the server and then start the test. I didn't see any way to boot the server as part of the Cypress config, like we did for Playwright.